jobs/e2e_runner.groovy

```diff
@@ -8,7 +8,7 @@ import utilities.StatusUpdater
   isMaster = config.type == 'master'
   isPR = config.type == 'pr'
   name = isMaster ? "e2e-runner" : "e2e-runner-${config.type}"
-  dockerPush = isPR ? 'docker-immutable-push' : 'docker-mutable-push'
+  dockerPush = 'docker-push'
```

Comment:
fixed in https://github.com/deis/jenkins-jobs/pull/108/commits/e21a13b7bde8f7c5d3bf43c2e1180f2ae8880836